test/utility/misc.c

```diff
@@ -139,6 +139,17 @@ testutil_cleanup(TEST_OPTS *opts)
 }
 
 /*
+ * testutil_check_long --
+ *	Check if WT_DISABLE_LONG_TESTS is set and exit if so
+ */
+void
+testutil_check_long(void)
```

Comment:
I don't love the name. "check" makes me think the function will return something.